chart/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+name: chart
```

Comment:
I believe this should be `estafette-cloudflare-dns`

Reply:
I had that but then I need to name the folder `estafette-cloudflare-dns` as well. Now I've used the Release.Name in the helpers. I don't like either, because I'd like to default to the chart name, but allow it to be customized as well if you want to run more than one of an application.